tests/runner.py

```diff
@@ -1354,30 +1354,40 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  def get_browser_open_func():
+    if not EMTEST_BROWSER:
+      print("Using default system browser")
+      return webbrowser.open_new
+    browser_args = shlex.split(EMTEST_BROWSER)
+    # If the given browser is just a type, use the built-in functionality
+    # of the webbrowser module to open that type of browser
+    if len(browser_args) == 1 and '/' not in browser_args[0]:
```

Comment:
`/` might be `os.path.sep` so it works on windows.

However, I'm a little unsure about this detection. Can we get a list of the "types" that are accepted by the API and check for them directly?

Reply:
Updated 👍

That may be possible? But webbrowser.open_new will just throw an exception if it's not a valid type.